### ISSUE TITLE
GC for MC 1.6.4 not working with the recommended Forge version

### ISSUE THREAD

**Issue by Robijnvogel** [micdoodle8/Galacticraft#2067]
It seems like Forge changed its own code to not work with its own values anymore.
This piece of code.
```
public static void registerHandlers()
	{
		TickRegistry.registerTickHandler(new GCCoreTickHandlerClient(), Side.CLIENT);
		NetworkRegistry.instance().registerChannel(new GCCorePacketHandlerClient(), GalacticraftCore.CHANNEL, Side.CLIENT);
		KeyBindingRegistry.registerKeyBinding(new GCCoreKeyHandlerClient());
	}
```
throws a NumberFormatException. Forge expects an integer or a string parseable to an integer as second parameter in the registerTickHandler, however Side.CLIENT's value is "minecraft" it seems.

This error was posted on:
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2358057-more-planets-mod-v-1-3-0-galacticraft-add-on?comment=742

**Comment by radfast** [comment]
Galacticraft 2 requires Forge 1147.
http://wiki.micdoodle8.com/wiki/Tutorials/Galacticraft_Installation_Guide

If it's not working with 1147 please re-open.

**Comment by Robijnvogel** [comment]
That Forge version is no longer available for download.
Probably because of an incompatibility with Java 8.

**Comment by radfast** [comment]
@Robijnvogel everything you have written is wrong.  Not just slightly wrong, but so badly wrong it's hard to imagine how you come up with it.

Please don't try to run before you can walk.

**Comment by Robijnvogel** [comment]
Hey sorry man,

In this particular case I could not find anything pointing to another mod in particular, especially because I ignored the Forge log that PowerBeastMan told me to ignore. And even if I hadn't, I wouldn't have had any clue it was an out-of date mod. (Actually, my theory is that PowerBeastMan found out by himself that his Chickenbones' mods were for 1.7.10 and he only got the Galacticraft crash report after he removed them. If not, he must have cut off the last part of his crash report, because CCC is not listed there. There is no other explanation.) I'll make sure that PowerBeastMan is only using Galacticraft and 4-Space before getting back to you (probably to apologize even more).

About your analogy: I'm not running. You are. I'm just standing on the sidelines watching, trying to understand the game, sometimes shouting something that helps, sometimes something that makes absolutely no sense. 
This is the reason why I filed this as an issue and not as a pull request. And it's probably the reason why you need to approve these pull requests and fix the issues before they get added to official releases.

**Comment by Robijnvogel** [comment]
I finally tested this myself (I should have done so way sooner, but I did not quite have the time) and I found out that everything runs for me without a problem.

It is not and definitely never was a 1.7.10 CCC problem. It's not a GC problem and it's not a 4-Space problem either.

I am assuming that he had Forge for 1.7.10 installed and he installed Forge 1.6.4 on top of it without removing the libraries folder. That explains the "Galacticraft expecting an item ID, but instead getting the "mod-segment" of a unique ID"-error.

I apologize once again for having been wrong and talking nonsense.
